src/main/java/org/fcrepo/importexport/ArgParser.java

```diff
@@ -130,6 +130,13 @@
                 .desc("When present this flag indicates that external content should be exported.")
                 .required(false).build());
 
+        // Retrieve inbound references
+        configOptions.addOption(Option.builder("i")
```

Comment:
This should not have any impact when importing — if the inbound references were exported, then they'll be found on disk and imported, if they weren't exported then they won't.

There are a couple of options that only make sense in either import or export contexts — with the general ability to reuse commands between import/export (and particularly with the config file), I don't think we should refuse to accept the options for the operations where they won't have any effect.  But maybe we should document this more clearly?  Maybe the CLI parser has some notion of option groups that would improve the help output, etc.?